dist/gcc-compatible/Hacl_Curve25519_64_Slow.c

```diff
@@ -66,32 +66,24 @@ static inline uint64_t add1_(uint64_t *out, uint64_t *f1, uint64_t f2)
 static inline void fadd_(uint64_t *out, uint64_t *f1, uint64_t *f2)
 {
   uint64_t c0 = (uint64_t)0U;
-  for (uint32_t i = (uint32_t)0U; i < (uint32_t)1U; i++)
   {
-    uint64_t t1 = f1[(uint32_t)4U * i];
-    uint64_t t20 = f2[(uint32_t)4U * i];
-    uint64_t *res_i0 = out + (uint32_t)4U * i;
+    uint64_t t1 = f1[(uint32_t)4U * (uint32_t)0U];
+    uint64_t t20 = f2[(uint32_t)4U * (uint32_t)0U];
+    uint64_t *res_i0 = out + (uint32_t)4U * (uint32_t)0U;
     c0 = Lib_IntTypes_Intrinsics_add_carry_u64(c0, t1, t20, res_i0);
-    uint64_t t10 = f1[(uint32_t)4U * i + (uint32_t)1U];
-    uint64_t t21 = f2[(uint32_t)4U * i + (uint32_t)1U];
-    uint64_t *res_i1 = out + (uint32_t)4U * i + (uint32_t)1U;
+    uint64_t t10 = f1[(uint32_t)4U * (uint32_t)0U + (uint32_t)1U];
```

Comment:
Using `0`(above)/`1` etc. instead of the index style (`(uint32_t)4U * (uint32_t)0U`) would be nicer. But I'd be fine with leaving it as is if it's too much work.

Reply:
krml doesn't have a CSE phase (normally this happens ahead of time anyhow in F*), I'm not against adding one, but this should be a separate issue... thanks! :-)